octobot_trading/exchanges/websockets/websockets_util.py

```diff
@@ -29,13 +29,21 @@ def check_web_socket_config(config, exchange_name) -> bool:
     return not force_disable_web_socket(config, exchange_name)
 
 
-def search_websocket_class(websocket_class, exchange_name):
+def search_websocket_class(websocket_class, exchange_name, tentacles_setup_config):
     for socket_manager in websocket_class.__subclasses__():
         # return websocket exchange if available
-        if socket_manager.has_name(exchange_name):
+        if socket_manager.has_name(exchange_name, tentacles_setup_config):
             return socket_manager
     return None
 
 
-def get_exchange_websocket_from_name(name):
-    return search_class_name_in_class_list(name, get_all_classes_from_parent(WebsocketExchange))
+def get_exchange_websocket_from_name(name, tentacles_setup_config):
+    for subclass in get_all_classes_from_parent(WebsocketExchange):
```

Comment:
Should we do a wrapper for this check in TentacleManager ? A method like `get_class_from_name_with_activated_tentacles`